docs/source/en/perf_infer_gpu_multi.md

```diff
@@ -15,11 +15,11 @@ rendered properly in your Markdown viewer.
 
 # Tensor parallelism in transformers
 
-[Tensor parallelism](./perf_train_gpu_many#tensor-parallelism) shards a model onto multiple GPUs and parallelizes computations such as matrix multiplication. It enables fitting larger model sizes into memory and is faster because each GPU can process a tensor slice.
+[Tensor parallelism](./perf_train_gpu_many#tensor-parallelism) shards a model onto multiple accelerators(e.g. CUDA GPU, Intel XPU) and parallelizes computations such as matrix multiplication. It enables fitting larger model sizes into memory and is faster because each accelerator can process a tensor slice.
```

Comment:
```suggestion
[Tensor parallelism](./perf_train_gpu_many#tensor-parallelism) shards a model onto multiple accelerators (CUDA GPU, Intel XPU, etc.) and parallelizes computations such as matrix multiplication. It enables fitting larger model sizes into memory and is faster because each accelerator can process a tensor slice.
```